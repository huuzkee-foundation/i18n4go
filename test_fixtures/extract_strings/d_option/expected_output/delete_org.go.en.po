# filename: d_option/input_files/org/delete_org.go, offset: 907, line: 32, column: 16
msgid "Delete an org"
msgstr "Delete an org"

# filename: d_option/input_files/org/delete_org.go, offset: 939, line: 33, column: 16
msgid "CF_NAME delete-org ORG [-f]"
msgstr "CF_NAME delete-org ORG [-f]"

# filename: d_option/input_files/org/delete_org.go, offset: 1025, line: 35, column: 35
msgid "Force deletion without confirmation"
msgstr "Force deletion without confirmation"

# filename: d_option/input_files/org/delete_org.go, offset: 1263, line: 42, column: 20
msgid "Incorrect Usage"
msgstr "Incorrect Usage"

# filename: d_option/input_files/org/delete_org.go, offset: 1541, line: 55, column: 44
msgid "org"
msgstr "org"

# filename: d_option/input_files/org/delete_org.go, offset: 1855, line: 71, column: 15
msgid "Org %s does not exist."
msgstr "Org %s does not exist."

# filename: d_option/input_files/org/delete_org.go, offset: 878, line: 31, column: 16
msgid "delete-org"
msgstr "delete-org"

# filename: d_option/input_files/org/delete_org.go, offset: 1491, line: 54, column: 13
msgid "f"
msgstr "f"

# filename: d_option/input_files/org/delete_org.go, offset: 1589, line: 60, column: 13
msgid "Deleting org %s as %s..."
msgstr "Deleting org %s as %s..."

