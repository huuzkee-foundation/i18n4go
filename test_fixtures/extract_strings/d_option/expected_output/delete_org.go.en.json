[{"id":"Delete an org","translation":"Delete an org"},{"id":"CF_NAME delete-org ORG [-f]","translation":"CF_NAME delete-org ORG [-f]"},{"id":"Force deletion without confirmation","translation":"Force deletion without confirmation"},{"id":"Incorrect Usage","translation":"Incorrect Usage"},{"id":"org","translation":"org"},{"id":"Org %s does not exist.","translation":"Org %s does not exist."},{"id":"delete-org","translation":"delete-org"},{"id":"f","translation":"f"},{"id":"Deleting org %s as %s...","translation":"Deleting org %s as %s..."}]
